tests/InputFiles/GrMhd/GhValenciaDivClean/BinaryNeutronStar.yaml

```diff
@@ -163,14 +189,37 @@ EventsAndTriggers:
           DelayChange: 0
           StepChoosers:
             - Cfl:
-                SafetyFactor: 0.5
+                SafetyFactor: &CflSafetyFactor 0.5
+  # Update time step size regularly.
+  - Trigger:
+      Slabs:
+        EvenlySpaced:
+          Interval: 100
+          Offset: 100
+    Events:
+      - ChangeSlabSize:
+          DelayChange: 20
+          StepChoosers:
+            - Cfl:
+                SafetyFactor: *CflSafetyFactor
+  - Trigger:
+      Slabs:
+        EvenlySpaced:
+          Interval: 50
+          Offset: 0
+    Events:
+      # BondiSachsInterpolation is for observing GW data. However, the
```

Comment:
This will technically still trigger even though there are zero events to run. Consider commenting out the trigger as well?